pkg/testing/open-match/mockpb/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
```

Comment:
Please update the year from 2022 to 2023.

Reply:
Thanks, did that.